docs/infrastructure/deployment-targets/cloud-target-discovery/index.md

```diff
@@ -32,6 +36,16 @@ To discover Azure cloud resources, Octopus uses the following variables:
 | ----------------------- | -------- | ------------------------------------------------------------------------------------------------------------- |
 | `Octopus.Azure.Account` | Y        | An [Azure account](/docs/projects/variables/azure-account-variables.md) to use when discovering cloud targets |
 
+From **Octopus 2022.3**, Azure steps that support target discovery will allow you to configure the variables above from within the step configuration if they have not been configured within your project yet.
+
+- Open the deployment process and navigate to the Azure step
+- Open the Cloud Connections section
+- Click the Azure tile. If target discovery variables have not been configured this will say "Configure Now".
+- Enter the credentials in the drawer
+- Click Ok. This will save the well-known variables above into your project.
```

Comment:
I think a numbered list works here better.

```suggestion
1. Open the deployment process and navigate to the Azure step.
1. Expand the **Cloud Connections** section.
1. Click the Azure tile. If target discovery variables have not been configured, this will say **Configure Now**.
1. Select the Azure Account to use.
1. Click **Ok**. This will save the well-known variables above into your project.
```

Also, just as an aside, does the Azure Account dropdown show accounts from the Infra tab, or a list of Azure Account variables? If it's the latter, it's interesting that we don't support (at least in the screenshot) binding to a variable like you can with the Account section for an Azure script step for example

Reply:
> I think a numbered list works here better.

Thanks, I'll update :) 

> does the Azure Account dropdown show accounts from the Infra tab or a list of Azure Account variables?

The dropdown only shows accounts from the infra tab. Saving the account sets the well-known variable `Octopus.Azure.Account` that is used for target discovery. We decided not to allow binding to an existing variable from within the drawer at the moment as the user flow is primarily targeted at newer users who want to discover cloud targets, so we wanted to keep it as simple as possible. Existing users already using variables to manage their accounts within steps are likely to already be a bit more advanced in their usage and so if they wanted to bind our well-known variable to a different one the intention is they can use the variables screen for that.

Reply:
Makes sense. Thank you for the detailed info